extism_pdk.go

```diff
@@ -203,21 +229,24 @@ func GetVar(key string) []byte {
 	return value
 }
 
+// `SetVar` sets the host variable associated with `key` to the `value` byte slice.
 func SetVar(key string, value []byte) {
 	keyMem := AllocateBytes([]byte(key))
 	defer keyMem.Free()
 
 	valMem := AllocateBytes(value)
 	defer valMem.Free()
 
-	extism_var_set(keyMem.offset, valMem.offset)
+	extismVarSet(keyMem.offset, valMem.offset)
 }
 
+// `GetVarInt` returns the int associated with `key` (or 0 if none).
```

Comment:
One other thing I was very tempted to change here was the return signature from `int` to `(int, bool)` to be more idiomatic Go (where the `bool` typically is assigned to a variable called `ok` and that is used to determine if the key actually exists)... but that would have been a breaking API change, so I left it alone.

If this were changed, then the signature for `GetVar` should probably also change, but I'm leaving them alone in this PR since I believe it is an orthogonal issue to the memory freeing issue.

Just FYI.

Reply:
Yeah that's a good point!